pkg/cmd/secret/list/list.go

```diff
@@ -21,20 +23,25 @@ type ListOptions struct {
 	HttpClient func() (*http.Client, error)
 	IO         *iostreams.IOStreams
 	BaseRepo   func() (ghrepo.Interface, error)
+	GetRepo    func(client *api.Client, repo ghrepo.Interface) (*api.Repository, error)
 
 	OrgName string
+	EnvName string
 }
 
 func NewCmdList(f *cmdutil.Factory, runF func(*ListOptions) error) *cobra.Command {
 	opts := &ListOptions{
 		IO:         f.IOStreams,
 		HttpClient: f.HttpClient,
+		GetRepo: func(client *api.Client, repo ghrepo.Interface) (*api.Repository, error) {
+			return api.GitHubRepo(client, repo)
+		},
 	}
 
 	cmd := &cobra.Command{
 		Use:   "list",
 		Short: "List secrets",
-		Long:  "List secrets for a repository or organization",
+		Long:  "List secrets for a repository, environment or organization",
```

Comment:
We prefer to use the Oxford comma:
```suggestion
		Long:  "List secrets for a repository, environment, or organization",
```

Reply:
Right on, I missed that in the guide